### ISSUE TITLE
"Show this query here again" checkbox does not work

### ISSUE THREAD

**Issue by barkermn01** [phpmyadmin/phpmyadmin#14144]
### Steps to reproduce
1. Login to PHP MyAdmin
2. Go to SQL Tab Write a none default query E.G `select * from table where 2=2`
3. Uncheck the "Show this query here again"
4. Submit query
4. go to SQL Tab again

### Expected behavior
The "Show this query here again" checkbox should remain unchecked and old query should have been replaced with the default `select * from table where 1`

### Actual behavior
It does not save the setting of "Show this query here again" checkbox so it's re-ticked and due to this auto-populates query box with the last run query

### Server configuration
**Operating system**:
Ubuntu 16.10

**Web server:**
Apache 2

**Database:**
MariaDB 10.2

**PHP version:**
PHP 7.2 FPM

**phpMyAdmin version:**
Version information: 4.9.0-dev, latest stable version: 4.7.9
Git revision: 695fe3a from master branch,
committed on Mar 28, 2018 at 01:48 PM by Weblate

### Client configuration
**Browser:**
Google Chrome - Version 64.0.3282.186 (Official Build) (64-bit)

**Operating system:**
Windows 10  v1709 - build 16299.248

**Comment by Piyush3079** [comment]
I have started working on this issue

**Comment by Piyush3079** [comment]
I have raised on issue related to this #14148 but need some discussion regarding this @ibennetch @mauriciofauth before solving this issue or that issue.

**Comment by Piyush3079** [comment]
I have checked the current stable. You are right, the auto save sql is not working there and no code is written for save this query in either stable or in current master, just a form is there.
I think in stable branch they are trying to get saved queries using get saved query button but in current master the implemented auto save sql feature doesn't require get saved query button as it always load saved query from local-storage.
I think the flow and logic of auto save sql needs to be changed from implementing show this query again.
Well although I have started working on this and will made a pr soon with the aforementioned features.
